### PR TITLE
Split linuxTPM -> {trousersTPM,wrappedTPM20}, implement CommandChannel in OpenConfig

### PR DESCRIPTION
**Pull request by twitchy-jsonp**
Refactor PR 2/2.

### CODE REVIEW

File: attest/tpm_linux.go [hunk 1]
```diff
@@ -118,281 +89,16 @@ func openTPM(tpm probedTPM) (*TPM, error) {
...
-			DigestAlg: alg.cryptoHash(),
-		}
-	}
-
-	return out, nil
-}
 
-func (t *linuxTPM) measurementLog() ([]byte, error) {
-	return ioutil.ReadFile("/sys/kernel/security/tpm0/binary_bios_measurements")
+	panic("unreachable")
```
Comment (ericchiang) [inline attest/tpm_linux.go:103]: This isn't unreachable right? It should return an error about being passed an unrecognized TPM version

File: attest/trousers_tpm_linux.go [hunk 2]
```diff
@@ -0,0 +1,149 @@
+// Copyright 2020 Google Inc.
```
Comment (ericchiang) [inline attest/trousers_tpm_linux.go:1]: Can we keep this in tpm_linux? I don't like having even more OS specific packages.

File: attest/tpm_linux.go (hunk 1 above)
Comment (twitchy-jsonp) [reply attest/tpm_linux.go:119]: It shouldnt be possible to set `version` to anything other than those two values without using unsafe (see code above, which populates `probedTPM.version`)

File: attest/trousers_tpm_linux.go (hunk 2 above)
Comment (twitchy-jsonp) [reply attest/trousers_tpm_linux.go]: Done

File: attest/tpm_linux.go (hunk 1 above)
Comment (twitchy-jsonp) [reply attest/tpm_linux.go:119]: Wait wut

Comment (twitchy-jsonp) [reply attest/tpm_linux.go:119]: Sec

Comment (twitchy-jsonp) [reply attest/tpm_linux.go]: Done. What the code said and what I thought it said were very different things ^^'

**Pull request merged**